x-pack/plugins/security_solution/public/overview/components/endpoint_notice/index.tsx

```diff
@@ -49,7 +49,7 @@ export const EndpointNotice = memo<{ onDismiss: () => void }>(({ onDismiss }) =>
         <EuiButton onClick={handleGetStartedClick} href={endpointsLink}>
           <FormattedMessage
             id="xpack.securitySolution.overview.endpointNotice.tryButton"
-            defaultMessage="Try Elastic Endpoint Security Beta"
+            defaultMessage="Try Elastic Endpoint Security (beta)"
```

Comment:
Not a blocker, but would it make sense to throw that `beta` badge here instead?

Reply:
good suggestion, we actually modeled our banner to be similar to the Ingest banner that's advertising the Elastic Agent.  This is something I'd like to sync up on.  Would be comfortable making that change before the final BC build.

We're using the beta badges on our actual pages link other places.